src/vl-rich-data-table.js

```diff
@@ -367,6 +412,23 @@ export class VlRichDataTable extends VlElement(HTMLElement) {
         });
     }
 
+    __observeFilterButtons() {
+        this.__filterCloseButton.addEventListener('click', () => {
+            this.hideFilterColumn();
+        });
+        this.__filterToggleButton.addEventListener('click', () => {
+            if (this.__searchColumnShown) {
+                this.hideFilterColumn();
+            } else {
+                this.showFilterColumn();
+            }
+        });
+    }
+
+    get __searchColumnShown() {
+        return Number(this.__searchColumn.getAttribute('size')) === VlRichDataTable._defaultSearchColumnSize;
```

Comment:
Aangezien het `hidden` attribuut gebruikt wordt (eerdere opmerking, is het veiliger om daarop te controleren.

Reply:
Ondertussen weggehaald door het attribuut te gebruiken